cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -799,6 +801,15 @@
 
       /** Default SSL keystore type */
       public static final String DEFAULT_SSL_KEYSTORE_TYPE = "JKS";
+
+      /** SSL truststore location */
+      public static final String SSL_TRUSTSTORE_PATH = "security.auth.server.ssl.truststore.path";
+      /** SSL truststore type */
+      public static final String SSL_TRUSTSTORE_TYPE = "security.auth.server.ssl.truststore.type";
+      /** SSL truststore key password */
+      public static final String SSL_TRUSTPASSWORD = "security.auth.server.ssl.truststore.keypassword";
```

Comment:
- What is this for, as it doesn't seem to be used? 
- Should it not be `SSL_TRUSTSTORE_KEY_PASSWORD` instead?